client/src/Overview/AddToCart.jsx

```diff
@@ -17,6 +17,8 @@ export default function AddToCart({ selectedStyle }) {
     const tempQuantityArray = [];
 
     setSelectedSize(e.target.value);
+    setSelectedQty(1);
```

Comment:
I'm not sure if there's exactly any issue, but this state is initially set as a string and then changed to a number. May want to keep consistent as number